### PR TITLE
Replace remaining print statements with log.*

### PR DESCRIPTION
**Pull request by hexylena**
All these things going unlogged... for those of us using sentry it's kinda scary.

### CODE REVIEW

File: lib/galaxy/util/__init__.py [hunk 1]
```diff
@@ -950,7 +950,7 @@ class DBNames( list ):
         man_builds = [(build, name) for name, build in man_builds]
         db_names = DBNames( db_names + man_builds )
     except Exception, e:
-        print "ERROR: Unable to read builds file:", e
+        log.error( "ERROR: Unable to read builds file: %s", str( e ) )
```
Comment (bgruening) [inline lib/galaxy/util/__init__.py]: the extra `str()` cast is not needed.

**Comment by nsoranzo** [comment]
A few flake8 errors need to be fixed.

**Comment by hexylena** [comment]
@nsoranzo yes, I'm aware. Just haven't gotten around to it yet.

**Comment by nsoranzo** [comment]
:+1:

**Comment by dannon** [comment]
+1

**Pull request merged**